packages/fluentui/docs/src/components/ComponentDoc/ComponentExample/ComponentExample.tsx

```diff
@@ -372,43 +356,49 @@ class ComponentExample extends React.Component<ComponentExampleProps, ComponentE
       },
     ];
 
-    return (
-      <Menu
-        primary
-        underlined
-        activeIndex={-1}
-        styles={codeEditorStyle}
-        variables={this.exampleMenuVariables}
-        items={menuItems}
-      />
-    );
+    return <Menu activeIndex={-1} styles={{ justifyContent: 'end' }} items={menuItems} />;
   };
 
   renderSourceCode = () => {
     const { currentCode = '', handleCodeChange } = this.props;
-    const lineCount = currentCode.match(/^/gm)!.length;
 
     return (
-      // match code editor background and gutter size and colors
-      <div style={{ background: EDITOR_BACKGROUND_COLOR } as React.CSSProperties}>
-        <div
-          style={
+      <Provider
+        theme={mergeThemes(
```

Comment:
Can you please create a them in root scope?

```tsx
const editorTheme = /* --- */

class ComponentExample {}
```

Otherwise it will a performance hit